ensime_shared/symbol_format.py

```diff
@@ -1,20 +1,20 @@
 # Funcions for symbols formatting
 
-def completion_to_suggest(completion):
+def completion_to_suggest(completion, formatted_type_params):
```

Comment:
It would be great to write a comment stating that `formatted_type_params` is a function.